.mdl_style.rb

```diff
@@ -5,3 +5,5 @@
 rule 'MD013', :line_length => 88
 # https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md029---ordered-list-item-prefix
 rule 'MD029', :style => :ordered
+# https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md024---multiple-headers-with-the-same-content
+rule "MD024", :allow_different_nesting => true
```

Comment:
This should be its own commit with an explanatory commit message.

Reply:
I created a new commit for the above change

Reply:
Please explain in the commit message why we need this change.

Reply:
I explained as following:
```
`markdownlint` check failed on CHANGELOG.md with error "Multiple
headers with the same content", because the `markdownlint` parameter
`allow_different_nesting` defaults to be `false`. To allow header
duplication under different nesting level, we need to explicitly set
`allow_different_nesting` to be true.
```

Reply:
I would prefer an explanation that gives the technical reason why `allow_different_nesting=True` is necessary. Using the default value of `False` as reason is not a convincing reason. It is similar to argue that a function was added because it was not there in the beginning.

A possible alternative would be, to give you an example:

Markdownlint requires each section headers to be different by default but the Changelog needs to contain multiple sections with the same name, therefore enable `allow_different_nesting` to support this use case.

I will not block this PR on this, though.

Reply:
I updated with the examples that you provide, thanks.